lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.h

```diff
@@ -472,6 +472,7 @@ class SwiftASTContext : public TypeSystemSwift {
     const DiagnosticCursor m_cursor;
 
   public:
+    enum class ErrorKind { swift_error, clang_error };
```

Comment:
since it's `enum class`, I think the cases could be shorter as `swift` and `clang`.